.github/workflows/ci.yml

```diff
@@ -119,6 +119,29 @@ jobs:
       - name: Run integration tests
         run: npm run test:integration
 
+  test-configs:
```

Comment:
Shall we add passing this suite as a precondition to `docker-main-branch` and `docker-versions-branch` and `publish-docs`? Also we probably want some consistency in our `needs:`; I see variations.